docs/tutorial/getting-started.md

```diff
@@ -31,34 +36,59 @@ Start off by deploying the jenkins-k8s charm. By default it will deploy the late
 of the jenkins-k8s charm.
 
 ```
-# Deploy an edge version of the charm until stable version is released.
 juju deploy jenkins-k8s --channel=latest/edge
 ```
 
+Wait for it to be active:
+```
+juju wait-for application jenkins-k8s
+```
+
 The Jenkins application can only have a single server unit. Adding more units through `--num-units`
 parameter will cause the application to misbehave.
 
-### Deploy and integrate agents
 
-By default, jenkins-k8s server application is installed with 0 executors for security purposes.
-A functional Jenkins application requires additional Jenkins agents to be integrated.
+### Expose jenkins-k8s through ingress
 
-The following commands deploy 3 units of the jenkins-agent-k8s charm and integrate them with the
-jenkins-k8s charm.
+Deploy traefik-k8s charm and integrate it with the jenkins-k8s charm:
+```
+juju deploy traefik-k8s --channel=latest/edge --trust
+juju integrate jenkins-k8s:ingress traefik-k8s
+```
 
+You can check the state with:
+```
+juju status --relations
 ```
```

Comment:
Maybe at this point say that the deployment is finished when all the units are in "active" status